### ISSUE TITLE
Export specific collection

### ISSUE THREAD

**Issue by AlexMuir** [KhronosGroup/glTF-Blender-IO#192]
I know there's been discussion on how to handle limiting the meshes being exported.  

I would love to see an option to select a single collection to export.  Since objects can belong in many collections, this would allow for users to select specific objects for export.  For example, I would create collections called 'LowPolyExport' and 'HighPolyExport'.

I know the current way to do this might be 'Export selected meshes' or even 'Export visible' but the problems with these are that they aren't persisted when the .blend is saved.  I want to have a collection so that I can repeat a glTF export from a .blend file without having to remember exactly what was selected last time.

I'm not up to speed on Blender's Python but I am diving into the source code at the moment to try and isolate issues. I have a very complex Blender scene which I'm trying to export to GLB and then across to Babylon JS - at the moment the exporter hangs constantly but I'm not able to share the scene publicly. So I am working to isolate the issues and create files that replicate them.  You'll see me here quite a bit for the next month :)

**Comment by AlexMuir** [comment]
I implemented this feature on a fork.  Forgive me - I never submitted a pull request, I don't really know Python, I have no idea how to automate any testing or anything like that so I'll just link to my code here.

I found that the FBX exporter is quite nicely set up and already included this feature.  I don't think the way I am doing it efficient at all - for every node I am parsing over the list of all objects in the active collection, converting it to a list and filtering based on that.  A much neater way, and the way that the FBX exporter does it, would be to simply select the active collection, get all its objects, and then pass them onto __filter_node() to be filtered.  

Anyway my effort at this is here.  

https://github.com/AlexMuir/glTF-Blender-IO/commit/bff2173bd21440e3e7bcf67135a37441dcbc08a8
